### ISSUE TITLE
Guide for editors

### ISSUE THREAD

**Issue by mwauke** [homermultitext/hmt-web#39]
The link that should lead to the editors guide on the homer multitext website is dead http://www.homermultitext.org/hmt-docs/HMTstyle-preview.pdf

http://homermultitext.github.io/hmt-editors-guide/editorial-policies/ is good, however

**Comment by StephanieML** [comment]
already taken care of it, and it will appear the next time the website is updated.

**Comment by neelsmith** [comment]
fixed?

**Comment by StephanieML** [comment]
yup, this was before we moved it to a github page, so definitely fixed
